Remotion/Web/Development.WebTesting/Utilities/DriverExtensions.cs

```diff
@@ -0,0 +1,83 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System.Collections.Generic;
+using Coypu;
+using Microsoft.Edge.SeleniumTools;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+
+namespace Remotion.Web.Development.WebTesting.Utilities
+{
+  /// <summary>
+  /// Extension methods for Coypu's <see cref="IDriver"/> interface.
+  /// </summary>
+  public static class DriverExtensions

```

Comment:
I think we can unit test these methods, meaning we instance the drivers, put them in an `IDriver` variable and call these methods without needing to start a browser session, please verify if that's possible and if yes, put the tests in our "Web.Development.WebTesting.UnitTests" project (I hope that is its name, didn't check)